build/media-suite_compile.sh

```diff
@@ -2464,7 +2550,10 @@ if [[ $mpv != n ]] && pc_exists libavcodec libavformat libswscale libavfilter; t
             done
         fi
 
-        unset mpv_ldflags replace PKGCONF_STATIC
+        LDFLAGS="$old_ldf"
+        replace=" ${mpv_cflags[*]} -Wno-int-conversion"
+        CFLAGS="${CFLAGS/$replace}"
+        unset mpv_ldflags replace PKGCONF_STATIC meson_opts default_lib old_ldf
```

Comment:
I'm not entirely happy with this cleanup, but it works:tm:
I'll look into `extra-cflags` options and similar for Meson